CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 Unreleased:
+- Updated JDK to Corretto 1.8.302
```

Comment:
Not a problem adding your change, but no one is actively keeping this changelog up to date. The last release in this changelog was when we first forked the repo.